hedera-node/test-clients/build.gradle.kts

```diff
@@ -271,30 +259,31 @@ tasks.register<Test>("testRepeatable") {
     modularity.inferModulePath.set(false)
 }
 
-tasks.shadowJar {
-    archiveFileName.set("SuiteRunner.jar")
+tasks.withType<ShadowJar>().configureEach {
+    group = "shadow"
+    from(sourceSets.main.get().output)
+    mergeServiceFiles()
 
-    manifest {
-        attributes(
-            "Main-Class" to "com.hedera.services.bdd.suites.SuiteRunner",
-            "Multi-Release" to "true"
-        )
-    }
+    manifest { attributes("Multi-Release" to "true") }
+
+    // There is an issue in the shadow plugin that it automatically accesses the
+    // files in 'runtimeClasspath' while Gradle is building the task graph.
+    // See: https://github.com/GradleUp/shadow/issues/882
+    dependencyFilter = NoResolveDependencyFilter()
 }
 
+application.mainClass = "com.hedera.services.bdd.suites.SuiteRunner"
```

Comment:
Doesn't moving this outside of the `shadowJar` task apply it to all jars produced from this file?

Reply:
Yes. But it's good that you ask so that I could double check.

This change is to align with other projects that are "applications" (use `org.hiero.gradle.module.application`).
`application.mainClass` is provided by Gradle's standard application plugin and we make sure to forward that to the Jar manifests here:
https://github.com/hiero-ledger/hiero-gradle-conventions/blob/main/src/main/kotlin/org.hiero.gradle.module.application.gradle.kts#L21-L22